tests/integration/mixer/policy/white_black_listing_test.go

```diff
@@ -23,7 +23,66 @@ import (
 	util "istio.io/istio/tests/integration/mixer"
 )
 
+func validateIPRule(t *testing.T, ctx framework.TestContext) {
+	t.Helper()
+	if galInst == nil {
+		t.Fatalf("galley not setup")
+	}
+	g := *galInst
+	if bookinfoNamespace == nil {
+		t.Fatalf("bookinfo namespace not allocated in setup")
+	}
+	bookinfoNs := *bookinfoNamespace
+	g.ApplyConfigOrFail(
+		t,
+		bookinfoNs,
+		bookinfo.NetworkingBookinfoGateway.LoadGatewayFileWithNamespaceOrFail(t, bookinfoNs.Name()))
+	g.ApplyConfigOrFail(
+		t,
+		bookinfoNs,
+		bookinfo.GetDestinationRuleConfigFile(t, ctx).LoadWithNamespaceOrFail(t, bookinfoNs.Name()),
+		bookinfo.NetworkingVirtualServiceAllV1.LoadWithNamespaceOrFail(t, bookinfoNs.Name()),
+	)
+	defer g.DeleteConfigOrFail(
```

Comment:
curious: does order matter in deletion? or could we collapse these two `DeleteConfigOrFail` calls?

Reply:
I modified the test to just use one config, since the virtual service "all v1" seems unnecessary.